locations/pipelines/apply_nsi_categories.py

```diff
@@ -68,9 +74,19 @@ def filter_cc(self, matches, cc) -> []:
             if "001" in include:  # 001 being global in NSI
                 globals_matches.append(match)
 
+        if categories:
+            # Find better match if category is supplied
+            includes = self.filter_categories(includes, categories)
+            globals_matches = self.filter_categories(globals_matches, categories)
+
         return includes or globals_matches
 
-    def filter_categories(self, matches, tags) -> []:
+    def filter_categories(self, matches: list[dict], tags: dict) -> list[dict]:
+        """Filter matches by category tags.
+        :param matches: list of matches from NSI
+        :param tags: category tags
```

Comment:
Can the documentation for `tags` clarify that categories have to be an exact match (e.g. if two tags are supplied, both tags have to present on the NSI item for a match to occur)?